third_party/terraform/data_sources/data_source_google_iam_policy.go

```diff
@@ -9,6 +9,36 @@ import (
 	"google.golang.org/api/cloudresourcemanager/v1"
 )
 
+var auditConfig *schema.Schema = &schema.Schema{
```

Comment:
Are these broken out for a good reason? I'd be inclined to inline them while you're here